### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by thomasthuynh**
Looking for feedback on my project 3!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,120 @@
...
+      const getLandingPageVideo = () => {
+        axios({
+          url: `https://api.themoviedb.org/3/movie/${movieData[0].id}/videos`,
+            params: {
+              api_key: "02a015f767f49fbd46124014022d6a5c",
+              append_to_response: "videos"
+            }
+        })
+          .then((res) => {
+              console.log(res.data.results);
```
Comment (esteraolszewski) [inline src/App.js]: Remove console.log

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,120 @@
...
+                setTrailer={setTrailer}
+              />
+            )
+          })}
+        </ul>
+      </main>
+
+      <footer>
+        <div className="wrapper">
+          <p>Created at <a href='https://junocollege.com/'>Juno College</a> using the MovieDB API</p>
```
Comment (esteraolszewski) [inline src/App.js:110]: Nice idea leaving a link to Juno College

**Review by esteraolszewski — COMMENTED** [review]
GREAT JOB 😊

File: src/App.js (hunk 1 above)
Comment (thomasthuynh) [reply src/App.js]: Ok, will remove.

**Pull request merged**